config/default.json

```diff
@@ -77,7 +77,9 @@
         "DisableLegacyMFA": true,
         "EnableEmailInvitations": false,
         "ExperimentalLdapGroupSync": false,
-        "ExperimentalStrictCSRFEnforcement": false
+        "ExperimentalStrictCSRFEnforcement": false,
+        "CreateBotAccounts": false,
+        "DisableBotsWhenOwnerIsDeactivated": true
```

Comment:
I noticed `EnableBotAccountCreation` is actually `CreateBotAccounts`, which is inconsistent with the model/config.go naming convention. Should this be changed?

Reply:
Yes. Ticket please. Needs to be 5.12.

Reply:
Ticket: https://mattermost.atlassian.net/browse/MM-15662